tidy3d/components/mode_spec.py

```diff
@@ -129,14 +129,14 @@ class ModeSpec(Tidy3dBaseModel):
         "yz plane, the ``bend_axis`` is always 1 (the global z axis).",
     )
 
-    bend_angle_rotation: bool = pd.Field(
+    angle_rotation: bool = pd.Field(
         False,
-        title="Use fields rotation when both bend and angle are defined",
-        description="Defines how modes are computed when both a bend and an angle are defined. "
-        "If `False`, the two coordinate transformations are directly composed. "
+        title="Use fields rotation when angle are defined",
+        description="Defines how modes are computed when an angle are defined. "
+        "If `False`, the coordinate transformations are directly composed. "
```

Comment:
If 'False', a coordinate transformation is applied through the permittivity and permeability tensors.